base.yml

```diff
@@ -15,6 +15,7 @@ services:
       - "BAT_MONGODB_URI=mongodb://mongo"
       - "BAT_POSTGRES_URL=postgres://eyeshade:password@eyeshade-postgres/eyeshade"
       - "BAT_REDIS_URL=redis://redis:6379"
+      - "BAT_RATIOS_URL=https://bat-ratio.herokuapp.com"
```

Comment:
we should create a new `basicattentiontoken.org` subdomain, the existing heroku app should be converted to our staging instance and a new production instance should be provisioned

Reply:
correct, just checked in with the devops issue: https://github.com/brave/devops/issues/230 (which I would assume would be this) can update when informed it is available.

Reply:
should be on terraform tomorrow. according to @brave-intl/devops